src/components/header/header.scss

```diff
@@ -80,3 +80,64 @@
     color: #312F33;
     cursor: pointer;
 }
+
+
+/* -------------- MEDIA QUERIES (MEDIUM DEVICES) -------------- */
+$breakpoint-tablet: 1024px;
+@media (max-width: $breakpoint-tablet) {
```

Comment:
make the breakpoint a global variable on a separate file, like that of color.scss, fonts.scss etc

Reply:
resolved 🙇